Makefile

```diff
@@ -263,6 +263,12 @@ gocyclo:
 	# Run gocyclo over all .go files
 	gocyclo -over 15 ${GOFILES}
 
+.PHONY: staticcheck
```

Comment:
https://github.com/aws/amazon-ecs-agent/blob/master/Makefile#L330 we have static-check already, can we name this differently? unused-check?

Reply:
I prefer staticcheck because it's the name of the tool we're using (https://github.com/dominikh/go-tools). We can't use unused-check because staticcheck does other checks that we will be adding soon.

If you'd prefer maybe I could do `go-tools-staticcheck` ?

Reply:
I put it as a command under the static-check target